thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/ThriftServiceEntry.java

```diff
@@ -73,6 +76,18 @@ public Set<Class<?>> interfaces() {
         return metadata.interfaces();
     }
 
+    /**
+     * Returns the associated thrift method name in proto from Thrift function.
+     *
+     * @return the associated thrift method name in proto, or {@code null} if there's no such function.
+     */
+    @Nullable
+    public String functionName(String method) {
+        return Optional.ofNullable(metadata.function(method))
```

Comment:
ditto. We can do this without `Optional`.

Reply:
👌🏻